channeldb/channel.go

```diff
@@ -161,6 +161,13 @@ type ChannelConstraints struct {
 	// acted upon in the case of a unilateral channel closure or a contract
 	// breach.
 	MaxAcceptedHtlcs uint16
+
+	// CsvDelay is the relative time lock delay expressed in blocks. Any
+	// settled outputs that pay to the owner of this channel configuration
+	// MUST ensure that the delay branch uses this value as the relative
+	// time lock. Similarly, any HTLC's offered by this node should use
+	// this value as well.
+	CsvDelay uint16
```

Comment:
Interesting... Does this work since `CsvDelay` is still serialized in the DB as part of the `ChannelConfig`?

Reply:
Yeah it works since we just [list out all the fields when we're writing the config, and this is embedded](https://github.com/lightningnetwork/lnd/blob/master/channeldb/channel.go#L2211). Probably could have been left out this PR all together, but works as is.